### ISSUE TITLE
Error 

### ISSUE THREAD

**Issue by dxfan227** [Kiina/MMM-trakt#19]
Hello I'm back. I had my Magic Mirror working but  unplugged it for a few months. Now it says error loading module check the logs. I can't seem to find where the problem is. @Kiina  do you have any ideas?

**Comment by Kiina** [comment]
Have you looked into the logs? Also did you try to delete the token.json and try to re-authenticate? My first instinct would be that trakt expires the token after a few month.

**Comment by dxfan227** [comment]
I’ll try to re-authorized.

This was my first “coding project” so I forgot how to do a lot of stuff lolz I looked through the terminal window as well as the pm2 logs and didn’t see any errors. We’re else would I look ?

**Comment by Kiina** [comment]
Yes, I think the pm2 logs should just mirror the node.js logs. Also if you have it unplugged for a long time, a quick git pull in the MMM-trakt directory could be helpful. There is improved logging so if deleting the token.json doesnt help, pull the newest version, add `debug: true` in the module part of your `MagicMirror/config/config.js` and post the full logs, maybe we can find something
